wordpress-mobile/WordPress-iOS
# [Stats Refresh] Update card header

Fixes #10498 

This actually encompasses these changes:
1) Updates the card header to the new design.
2) Removes the full border on the cards, and now has only a top and bottom.
3) Updates margins for the Latest Post Summary text.
4) Fixes an issue where All Time Stats rows were being duplicated on pull to refresh.

The new card header should match this Zeplin design zpl://screen?pid=5a86dd8dfaff0fa0290982db&sid=5bf2a8d9e6b0427eb9a22108 (Full Stats Refresh > Heading Bar Changes).

To test:

---
On a site with Latest Post Summary data, Go to Stats > Insights. Verify the view is:

![with_lps](https://user-images.githubusercontent.com/1816888/48814222-2d8b9600-ecf7-11e8-827b-0032040ec307.png)

---
On a site with _no_ Latest Post Summary data, Go to Stats > Insights. Verify the view is:

![no_lps](https://user-images.githubusercontent.com/1816888/48814247-50b64580-ecf7-11e8-9960-8b94bcf87bd8.png)

---
On a site with _no_ All Time Stats data, Go to Stats > Insights. Verify the view is:

![no_ats](https://user-images.githubusercontent.com/1816888/48814672-e1d9ec00-ecf8-11e8-8627-ef836dbe1638.png)
